### ISSUE TITLE
🐛 Incorrect Mog house Model in Southern San d'Oria [S]

### ISSUE THREAD

**Issue by N3ckB3ard** [DerpyProjectGroup/topaz#3987]
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my issue will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated.

## Branch affected by issue 

`base`

## Steps to reproduce

!zone 80
Enter MH behind you
![image](https://github.com/LandSandBoat/server/assets/98081508/28aabdb1-5042-45a0-b6b0-dd114bfd3184)

## Expected behavior

Enter MH zone with correct model

**Comment by Mortalelite** [comment]
I have tested this myself and get same results as posted in the image. Bastok and Windurst [S] you do get MH model

**Comment by TeoTwawki** [comment]
_mog slaps entry way_   
"This baby can fit so much furniture in the hell void!" 

https://github.com/LandSandBoat/server/blob/6834a187dc8042d3a351ded5f6fa24584897fb0b/src/map/packets/zone_in.cpp#L33

So we need to redo packet caps for entering mog houses in general, because its not clear wtf is going on. The value in that spot in the packet can't simply be zone ID because the other mogs aren't bugged like this and their values match other zone IDs, Yet this one matches rakaznar u3 and we got rakaznar u3's model loading here. And as funny as that is, it makes no sense for it to magically be the id of that model for 1 mog and not any of the others.

```cpp
    switch (zoneutils::GetCurrentRegion(PChar->getZone()))
    {
        case REGION_TYPE::WEST_AHT_URHGAN:
            return 214;
        case REGION_TYPE::RONFAURE_FRONT:
            return 189;
        case REGION_TYPE::GUSTABERG_FRONT:
            return 199;
        case REGION_TYPE::SARUTA_FRONT:
            return 219;
        case REGION_TYPE::SANDORIA:
            return PChar->profile.nation == 0 ? 0x0121 : 0x0101;
        case REGION_TYPE::BASTOK:
            return PChar->profile.nation == 1 ? 0x0122 : 0x0102;
        case REGION_TYPE::WINDURST:
            return PChar->profile.nation == 2 ? 0x0123 : 0x0120;
        case REGION_TYPE::JEUNO:
            return 0x0100;
        case REGION_TYPE::ADOULIN_ISLANDS:
            return 0x0124;
        default:
            ShowWarning("Default case reached for GetMogHouseID by %s (%u)", PChar->GetName(), PChar->getZone());
            return 0x0100;
```
also 0x0100 would be west adoulin if these were zone IDs. Jeuno mog isn't loading west adoulin!

Bottom line: we're missing something about what these values are and do. Not just the 1 we're using, but in that packet in general.
